### ISSUE TITLE
Request: clarification of interaction between quant options --pseudobam and --genomebam

### ISSUE THREAD

**Issue by SolKatzman** [pachterlab/kallisto#367]
kallisto v0.48.0

The documentation of the 2 kallisto quant options: --pseudobam and --genomebam is a bit ambiguous.

a) are they mutually exclusive?  If so an error should be thrown if both are specified, which does not happen
b) is genomebam a sub-option of pseudobam ? -- maybe, but genomebam works standalone
c) something else, like the order matters?

Regardless of which is specified, the output bam file has the same name, which is also confusing
(and not specified for the genomebam option.

Explicit documentation would save trial-and-error work.

/Sol Katzman

**Comment by Yenaled** [comment]
Yes, they are mutually exclusive. We are still in the process of updating genomebam/pseudobam, but it's mostly an auxiliary feature that isn't part of standard kallisto use.

genomebam works as stand-alone and projects the mappings to genome coordinates. pseudobam only outputs a bam file based on transcript one (e.g. impossible to determine chromosomal location)..
